### PR TITLE
Day 12

### PR DESCRIPTION
**Pull request by ylruiz**
Login with social services (facebook and google)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,24 +1,19 @@
...
- *  For more info see: https://github.com/expressjs/body-parser
- */
 const bodyParser = require('body-parser');
+const passport = require('./config/passport');
+const cookieParser = require('cookie-parser');
 
 // create your app
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 5000;
```
Comment (krvajal) [inline app.js]: Why this port change?

File: app.js [hunk 2]
```diff
@@ -27,12 +22,25 @@ app.set('view engine', 'mustache')
...
+app.use(passport.session());
+
+/*** END CONFIGURE ***/
+
+/*** START DEFINE ROUTES ***/
 
 // GET root
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index', {welcome: ""});
```
Comment (krvajal) [inline app.js]: Lets move this into a `home.js` controller

File: app.js (hunk 1 above)
Comment (ylruiz) [reply app.js]: I changed the port because I am using the 3000 port in another application.

Comment (krvajal) [reply app.js]: What does it even mean?
Why this code should even care about the existence of another application. If you want to run in in your machine in a different port. Then add the PORT env variable to your env file and that it. You dont need to change the code adapt to your local environment requirements

**Comment by krvajal** [comment]
the PR was never approved ;(

**Comment by ylruiz** [comment]
Sorry, I thought that it was ready because you changed the Day 12 track in
Trello to the done list. 😳

**Pull request merged**